packages/@aws-cdk/aws-appmesh/lib/virtual-router.ts

```diff
@@ -174,7 +181,7 @@ export class VirtualRouter extends VirtualRouterBase {
 /**
  * Interface with properties ncecessary to import a reusable VirtualRouter
  */
-interface VirtualRouterAttributes {
+export interface VirtualRouterAttributes {
```

Comment:
If that becomes exported, but was not before... Should it be explicitly marked `@experimental`? Or is the whole module experimental?

Reply:
The module is experimental. I also added `fromAttributes` which is new, needs this type to be public.